tests/TestCase.php

```diff
@@ -173,16 +173,16 @@ public function checkReportbackResponse($response)
     }
 
     /**
-     * Create a reaction request.
+     * Create a reaction.
      *
      * @return array
      */
-    public function createReactionRequest($northstarId, $photoId, $postType)
+    public function createReaction($northstarId, $photoId, $postType)
     {
-        return [
+        $this->json('POST', $this->reactionsApiUrl, [
```

Comment:
I'm still not convinced of the value of abstracting this into a method on the TestCase, especially since this would error out if called from anywhere besides `ReactionsApiTest` (since `$this->reactionsApiUrl` wouldn't be defined).